### ISSUE TITLE
"command-background": "white" is not white

### ISSUE THREAD

**Issue by mstorus** [tldr-pages/tldr-node-client#56]
I'm trying to make tldr to look good on the default Mac OSX Terminal white background, and tried setting `command-background` to `white` in `~/.tldrrc`. But it seems like the rendered color is not actually white.

![210x50-1](https://cloud.githubusercontent.com/assets/1182738/12103063/317ee900-b2f4-11e5-94ba-639992d40d64.jpg)

**Comment by agnivade** [comment]
We have an ongoing issue on proper themes support. Background colors will be properly supported then.

Till that time, you can do this in your `.tldrrc` to achieve your purpose - 
```json
{
  "themes": {
    "original": {
      "name": "bold",
      "description": "", 
      "exampleDescription": "green",
      "exampleCode": "red,bgWhite",
      "exampleArgument": "black,bgWhite"
    }   
  },  
  "theme": "original"
}
```

**Comment by agnivade** [comment]
Fixed by #117
